### PR TITLE
Add wheel

### PR DESCRIPTION
**Pull request by jakirkham**
Adds a recipe for `wheel` written from scratch. Please provide a careful review of this so we can get it right. Also more maintainers welcome. 😄 

cc @conda-forge/core

### CODE REVIEW

File: recipes/wheel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{% set name = "wheel" %}
+{% set version = "0.29.0" %}
+{% set checksum = "555a67e4507cedee23a0deb9651e452f" %}
+{% set number = 0 %}
```
Comment (pelson) [inline recipes/wheel/meta.yaml]: Would you mind removing this line?

Comment (jakirkham) [inline recipes/wheel/meta.yaml]: Yeah, I was meaning to actually use it in the recipe. Though I have no problem dropping it.

Comment (jakirkham) [inline recipes/wheel/meta.yaml]: Done.

**Comment by pelson** [comment]
I'm happy to be added as a maintainer in the feedstock.

**Pull request merged**